tests/isar/state_machine/test_state_machine.py

```diff
@@ -753,3 +753,61 @@ def _mock_robot_exception_with_message() -> RobotException:
         error_reason=ErrorReason.RobotUnknownErrorException,
         error_description="This is an example error description",
     )
+
+
+def test_transition_from_monitor_to_pausing(
+    sync_state_machine: StateMachine,
+) -> None:
+    sync_state_machine.mission_ongoing = True
+    task_1: Task = TakeImage(
+        target=DummyPose.default_pose().position, robot_pose=DummyPose.default_pose()
+    )
+    sync_state_machine.current_task = task_1
+    sync_state_machine.current_mission = Mission(name="Dummy misson", tasks=[task_1])
+    sync_state_machine.state = sync_state_machine.monitor_state.name  # type: ignore
+
+    monitor_state: EventHandlerBase = cast(
+        EventHandlerBase, sync_state_machine.monitor_state
+    )
+    event_handler: Optional[EventHandlerMapping] = (
+        monitor_state.get_event_handler_by_name("pause_mission_event")
+    )
+
+    assert event_handler is not None
+
+    event_handler.event.trigger_event(True)
+    transition = event_handler.handler(event_handler.event)
+
+    assert transition is sync_state_machine.pause  # type: ignore
+
+    transition()
+    assert sync_state_machine.state is sync_state_machine.pausing_state.name  # type: ignore
+
+
+def test_transition_from_returning_home_to_pausing_return_home(
```

Comment:
It could be good with tests from pausing/pausing_returning_home to paused/return_home_paused as well, just so we don't forget to add them in the future.